src/pages/api/graphql/resolvers.ts

```diff
@@ -0,0 +1,20 @@
+import { Column } from '../models/column';
+import { Task } from '../models/task';
+import ColumnRepository from '../repositories/columnRepository';
+import TaskRepository from '../repositories/taskRepository';
+
+const columnRepo = new ColumnRepository();
+const taskRepo = new TaskRepository();
+
+const resolvers = {
```

Comment:
Move this file away from api folder. Everything inside that can be called by a client.